arrows/vtk/applets/fuse_depth.cxx

```diff
@@ -418,14 +412,9 @@ class fuse_depth::priv
           inPts->GetPoint(i, points[i].data());
         }
 
-        kwiver::arrows::pdal::pointcloud_io pc_io;
-        pc_io.set_local_geo_cs(lgcs);
-        pc_io.save(output_mesh_file, points);
-#else
-        throw vital::file_write_exception(output_mesh_file,
-                                          "KWIVER was not compiled with PDAL, "
-                                          "cannot write LAS.");
-#endif
+        auto pc_io = vital::algo::pointcloud_io::create("pdal");
+        pc_io->set_local_geo_cs(lgcs);
```

Comment:
We need a check here that `pc_io` was created successfully and is not null before we try to use it.  You can thrown an exception if `pc_io` is null.